app/src/androidTest/java/com/github/epfl/meili/NewPostTitleAndTextTest.kt

```diff
@@ -0,0 +1,193 @@
+package com.github.epfl.meili
+
+
+import android.util.Log
+import android.view.View
+import android.view.ViewGroup
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.IdlingRegistry
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import com.github.epfl.meili.forum.ForumActivity
+import com.github.epfl.meili.forum.ForumCountingIdlingResource
+import com.github.epfl.meili.forum.NewPostActivity
+import com.github.epfl.meili.forum.PostActivity
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.TypeSafeMatcher
+import org.hamcrest.core.IsInstanceOf
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
+class CanCreateNewPostTest {
+
+    @Rule
+    @JvmField
+    var mActivityTestRule = ActivityTestRule(MainActivity::class.java)
+
+    @Test
+    fun canCreateNewPostTest() {
```

Comment:
Do you think it would be a good idea to split this test so that if it ever fails we can immediately identify what is failing from the test name? And maybe if it makes sense add some assertions every time you perform an action like clicking?